### ISSUE TITLE
4.2: Crash after login

### ISSUE THREAD

**Issue by st0rmchild** [gitbucket/gitbucket#1225]
## Issue
**Impacted version**: 4.2.0

**Deployment mode**: *standalone app*

**Problem description**:
I have just updated to the latest version. After the login I get the following error:

```
org.h2.jdbc.JdbcSQLException: Column "X21.ALLOW_WIKI_EDITING" not found; SQL statement:
CREATE FORCE VIEW PUBLIC._3 AS
SELECT
    X21.UPDATED_DATE AS X22,
    X21.ALLOW_WIKI_EDITING AS X23,
    X21.EXTERNAL_ISSUES_URL AS X24,
    X21.PARENT_REPOSITORY_NAME AS X25,
    X21.EXTERNAL_WIKI_URL AS X26,
    X21.DEFAULT_BRANCH AS X27,
    X21.DESCRIPTION AS X28,
    X21.ORIGIN_REPOSITORY_NAME AS X29,
    X21.ORIGIN_USER_NAME AS X30,
    X21.PARENT_USER_NAME AS X31,
    X21.PRIVATE AS X32,
    X21.REPOSITORY_NAME AS X33,
    X21.USER_NAME AS X34,
    X21.ENABLE_WIKI AS X35,
    X21.ENABLE_ISSUES AS X36,
    X21.REGISTERED_DATE AS X37,
    X21.LAST_ACTIVITY_DATE AS X38
FROM PUBLIC.REPOSITORY X21 [42122-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Alias.optimize(Alias.java:51)
	at org.h2.command.dml.Select.prepare(Select.java:852)
	at org.h2.command.Parser.prepare(Parser.java:239)
	at org.h2.engine.Session.prepare(Session.java:525)
	at org.h2.engine.Session.prepare(Session.java:512)
	at org.h2.table.TableView.compileViewQuery(TableView.java:105)
	at org.h2.table.TableView.initColumnsAndTables(TableView.java:156)
	at org.h2.table.TableView.init(TableView.java:98)
	at org.h2.table.TableView.<init>(TableView.java:65)
	at org.h2.table.TableView.createTempView(TableView.java:534)
	at org.h2.command.Parser.readTableFilter(Parser.java:1192)
	at org.h2.command.Parser.readJoin(Parser.java:1549)
	at org.h2.command.Parser.parseJoinTableFilter(Parser.java:1941)
	at org.h2.command.Parser.parseSelectSimpleFromPart(Parser.java:1897)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:2044)
	at org.h2.command.Parser.parseSelectSub(Parser.java:1890)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:1711)
	at org.h2.command.Parser.readTableFilter(Parser.java:1181)
	at org.h2.command.Parser.parseSelectSimpleFromPart(Parser.java:1896)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:2044)
	at org.h2.command.Parser.parseSelectSub(Parser.java:1890)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:1711)
	at org.h2.command.Parser.parseSelect(Parser.java:1699)
	at org.h2.command.Parser.parsePrepared(Parser.java:445)
	at org.h2.command.Parser.parse(Parser.java:317)
	at org.h2.command.Parser.parse(Parser.java:289)
	at org.h2.command.Parser.prepareCommand(Parser.java:254)
	at org.h2.engine.Session.prepareLocal(Session.java:560)
	at org.h2.engine.Session.prepareCommand(Session.java:501)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1202)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:73)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:679)
	at com.zaxxer.hikari.pool.ProxyConnection.prepareStatement(ProxyConnection.java:323)
	at com.zaxxer.hikari.pool.HikariProxyConnection.prepareStatement(HikariProxyConnection.java)
	at scala.slick.jdbc.JdbcBackend$SessionDef$class.prepareStatement(JdbcBackend.scala:152)
	at scala.slick.jdbc.JdbcBackend$BaseSession.prepareStatement(JdbcBackend.scala:389)
	at scala.slick.jdbc.StatementInvoker.results(StatementInvoker.scala:32)
	at scala.slick.jdbc.StatementInvoker.iteratorTo(StatementInvoker.scala:22)
	at scala.slick.jdbc.Invoker$class.foreach(Invoker.scala:64)
	at scala.slick.jdbc.StatementInvoker.foreach(StatementInvoker.scala:16)
	at scala.slick.jdbc.Invoker$class.build(Invoker.scala:47)
	at scala.slick.jdbc.StatementInvoker.build(StatementInvoker.scala:16)
	at scala.slick.jdbc.Invoker$class.list(Invoker.scala:39)
	at scala.slick.jdbc.StatementInvoker.list(StatementInvoker.scala:16)
	at gitbucket.core.service.ActivityService$class.getRecentActivitiesByOwners(ActivityService.scala:47)
	at gitbucket.core.controller.IndexController.getRecentActivitiesByOwners(IndexController.scala:13)
	at gitbucket.core.controller.IndexControllerBase$class.gitbucket$core$controller$IndexControllerBase$class$$$anonfun$3(IndexController.scala:52)
	at org.scalatra.ScalatraBase$class.org$scalatra
```

**Comment by takezoe** [comment]
@st0rmchild Could you show records in `VERSIONS` table in your database?

**Comment by takezoe** [comment]
Merged with #1227
